examples/template-ts/package.json

```diff
@@ -1,15 +1,18 @@
 {
-  "version": "0.0.0",
+  "name": "feedback-template",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.html",
   "scripts": {
-    "dev": "vite",
-    "build": "tsc && vite build",
-    "serve": "vite preview"
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "parcel-bundler": "1.12.5",
+    "xstate": "^4.32.1"
   },
   "devDependencies": {
     "typescript": "^4.7.4",
     "vite": "^2.9.13"
```

Comment:
you have switched to using parcel for this example so vite should be dropped from here